raspi/soundserver/soundserver.py

```diff
@@ -3,12 +3,16 @@
 from werkzeug.security import generate_password_hash, check_password_hash
 import sys
 import os
+import subprocess
+import logging
+import os
 from vlcplayer import vlcplayer
 
 app = Flask(__name__)
 
+logger = logging.getLogger(__name__)
 dir_path = os.path.dirname(os.path.realpath(__file__))
-
+song_path = "/home/pi/Music"
```

Comment:
We do not intend to store music on the device itself, search music in mounted devices.

Reply:
changed in the new commit